app/views/crops/_thumbnail.html.haml

```diff
@@ -1,13 +1,14 @@
-.thumbnail
-  .crop-thumbnail
-    - if crop
-      = link_to image_tag((crop.default_photo ?  crop.default_photo.thumbnail_url : 'placeholder_150.png'), :alt => crop.name, :class => 'img'), crop
-      .cropinfo
-        .cropname
-          = link_to crop.name, crop
-        - if crop.scientific_names.size > 0
-          .scientificname
-            = crop.scientific_names.first.scientific_name
-        .plantingcount
-          Planted
-          = pluralize(crop.plantings.size, "time")
+- cache cache_key_for(Crop) do
```

Comment:
This keys the fragment based on the Crop class - it's expired when any crop is updated or added, and I think in the meantime it will return the same fragment for *any* crop. We should base the cache key on the individual crop object, or at least pass in a value to the `identifier` keyword argument.